### PR TITLE
[PFCWD]: Periodically poll WD counters

### PR DESCRIPTION
**Pull request by marian-pritsak**
For every action handler call commitCounters on a periodic basis

Signed-off-by: marian-pritsak <marianp@mellanox.com>

### CODE REVIEW

File: orchagent/pfcwdorch.cpp [hunk 1]
```diff
@@ -22,6 +22,7 @@
 #define PFC_WD_POLL_TIMEOUT             5000
 #define SAI_PORT_STAT_PFC_PREFIX        "SAI_PORT_STAT_PFC_"
 #define PFC_WD_TC_MAX 8
+#define COUNTER_CHECK_POLL_TIMEOUT_SEC  5
```
Comment (lguohan) [inline orchagent/pfcwdorch.cpp]: prefer 1 second

Comment (nikos-li) [reply orchagent/pfcwdorch.cpp]: Is the polling interval controlled via cli or via config_db.json? If yes can you please explain how?

If not this needs fixing prior to committing.

Comment (lguohan) [reply orchagent/pfcwdorch.cpp]: this is not controllable from the cli, this is only polling queue when the queue in pfc recovery state, in this case, we need to periodically poll it to get dropped packets. I do not see a generic need to make it controllable from cli.

Comment (nikos-li) [reply orchagent/pfcwdorch.cpp]: @lguohan All polling intervals need to be controlled via the cli and should provide the ability to be disabled.

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
@nikos-li , this is a internal feature and the user should not care about it and does not need to control that.

**Pull request merged**